quiz/web/src/modules/common/components/Start/Start.tsx

```diff
@@ -0,0 +1,69 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import TextField from "@mui/material/TextField";
+import Typography from "@mui/material/Typography";
+import NavBar from "modules/common/components/Start/NavBar";
+import CachedIcon from "@mui/icons-material/Cached";
+import IconButton from "@mui/material/IconButton";
+import InputAdornment from "@mui/material/InputAdornment";
+
+function Start() {
+    return (
+        <div className="w-screen h-screen">
+            <NavBar />
+            <Container
+                maxWidth="xl"
+                sx={{ display: "flex", flexDirection: "column", alignItems: "center", my: 5, alignContent: "center" }}
```

Comment:
double check if this part cannot be done with Tailwind
```className="flex flex-col items-center content-center my-5"```

Reply:
<img width="1728" alt="screen" src="https://user-images.githubusercontent.com/102250199/229290591-4fb5cfa6-8656-40ef-aa6e-1b22b6e8a927.png">

I checked on all MUI components if it is possible to overwrite styles with Tailwind. From my combinations it turned out that it is not possible to overwrite in this way, a simpler way to overwrite MUI styles with "sx" (from MUI documentation)
On the screen below, I marked that I entered the line of code you suggested, but it does not work, unlike "sx"
It could be, of course, that I missed something. Have you tried doing this on yours?

Reply:
Yup seems like a bit of tweaking to make this work will be needed, it's fine as it was then 😄

Reply:
fixed